documentation/modules/upgrading/proc-upgrade-cluster-operator.adoc

```diff
@@ -37,14 +37,29 @@ kubectl apply -f install/cluster-operator
 +
 Wait for the rolling updates to complete.
 
+. If the new Operator version no longer supports the Kafka version you are upgrading from, an error message is returned by the Cluster Operator. For example:
++
+[source,shell,subs="+quotes,attributes"]
+----
+"Version {KafkaVersionLower} is not supported. Supported versions are: #...".
```

Comment:
`{KafkaVersionLower}` is technically still supported so the message might be confusing. We should either add a new variable for the _unssuported_ version or just hardcode here something ... for example `2.4.0`.

Reply:
I hardcoded the error message for Kafka 2.4.0. That's much simpler than using variables.